_posts/2022-04-12-identity.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title:  "How Flotta project proofs identity on the far edge"
+date:   2022-04-12 12:00:00 +0100
+categories: flotta
+author: Eloy Coto <eloy.coto@acalustra.com>
+tags:
+  - authentication
+  - flotta
+
+summary-1: How Flotta project proofs identity on the far edge
+---
+
+When speaking about the edge one of the most problematic thing is the security.
+At the edge, you cannot assume there is security at all, so we assume that all
+your devices are compromised by default and we adopt a [Zero
+trust](https://cloud.google.com/beyondcorp) approach.
+
+Edge Authentication is getting traction, and at the moment, there are multiple
+projects out there that take care of it. For us, the most promising one, is the
+protocol-driven by the [FIDO alliance](https://fidoalliance.org/), regarding
```

Comment:
WDYT about "The most promising one to us is FDO - protocol-driven Device Onboarding project by FIDO alliance."